src/Arcus.Security.Providers.AzureKeyVault/Configuration/SecretStoreBuilderExtensions.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System.Security.Cryptography.X509Certificates;
+using Arcus.Security.Providers.AzureKeyVault;
+using Arcus.Security.Providers.AzureKeyVault.Authentication;
+using Arcus.Security.Providers.AzureKeyVault.Configuration;
+using GuardNet;
+
+// ReSharper disable once CheckNamespace
+namespace Microsoft.Extensions.Hosting
+{
+    /// <summary>
+    /// Extensions on the <see cref="SecretStoreBuilder"/> to provide easy addition the Azure Key Vault secrets in the secret store.
+    /// </summary>
+    public static class SecretStoreBuilderExtensions
+    {
```

Comment:
Nit - We might want to update our summaries, or use remarks, to talk about the auth mechanism per operation?

Reply:
Yes, added some.